plugins/colored-man-pages/colored-man-pages.plugin.zsh

```diff
@@ -15,6 +15,7 @@ less_termcap[se]="${reset_color}"
 # underlining
 less_termcap[us]="${fg_bold[green]}"
 less_termcap[ue]="${reset_color}"
+export GROFF_NO_SGR=1
```

Comment:
Although this patch solves the issue, exporting this here will change the environment for the user, which is something that might be unexpected, a solution which doesn't change the environment follows:

```diff
diff --git a/plugins/colored-man-pages/colored-man-pages.plugin.zsh b/plugins/colored-man-pages/colored-man-pages.plugin.zsh
index 981992d8..57facbb5 100644
--- a/plugins/colored-man-pages/colored-man-pages.plugin.zsh
+++ b/plugins/colored-man-pages/colored-man-pages.plugin.zsh
@@ -36,6 +36,7 @@ function colored() {
   # Prefer `less` whenever available, since we specifically configured
   # environment for it.
   environment+=( PAGER="${commands[less]:-$PAGER}" )
+  environment+=( GROFF_NO_SGR=1 )
 
   # See ./nroff script.
   if [[ "$OSTYPE" = solaris* ]]; then
```